ethportal-api/src/types/distance.rs

```diff
@@ -38,6 +38,15 @@ impl Distance {
         self.0.to_big_endian(&mut be);
         be
     }
+
+    /// Returns the top 4 bytes representation of `self`.
+    pub fn big_endian_u32(&self) -> u32 {
```

Comment:
I think this can be simplified:


    pub fn big_endian_u32(&self) -> u32 {
        let be_bytes: [u8; 4] = self.big_endian()[..4]
            .try_into()
            .expect("to get top 4 bytes from distance");
        u32::from_be_bytes(be_bytes)
    }